R/model.R

```diff
@@ -1,15 +1,54 @@
 #' @title Run the simulation
 #' @description
-#' The main entrypoint for the simulation. run_simulation puts together the
-#' model components and runs the malaria simulation. This currently returns a
```

Comment:
I like the line "This currently returns a dataframe with the number of individuals in each state at each timestep."